templates/workflow/stakeholder_list.html

```diff
@@ -78,7 +78,7 @@ <h4 class="page-title">Stakeholder List</h4>
       </div>
 
       <div class="sub-nav-item">
-        <div class="btn-group" role="group" aria-label="...">
+        <div id="add_stakeholder_modal" class="btn-group" role="group" aria-label="...">
```

Comment:
@IvyMMutiso one last question are you adding stakeholder modal CRUD in this PR as well?

Reply:
id deleted

Reply:
thanks Ivy :)